static-website-aws-python/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /usr/local/bin
```

Comment:
I wouldn't commit the `pyvenv.cfg` file. It generally isn't something we send in a repo (it lives in the `venv` directory if you're using the built-in virtual environment tool, which doesn't get committed), and someone will wonder why it appears. Pulumi creates the virtual environment based on the local Python environment, instead, and this file doesn't force it to use a specific version or anything.

Reply:
Ah, good spot! I actually had a line in my code to delete these files, but it had a typo, and I didn't notice them sneaking through. Will remove. Thanks!